src/components/books.jsx

```diff
@@ -0,0 +1,7 @@
+const Books = () => (
+  <>
```

Comment:
- [ ] Great job with the navigation. please remember the project guidline says your home page should contain a list of books and author plus a form to create them. 
-- I suggest you arrange your work with a `newbookform`,` booklist` and `book` component under the books page component where by the booklist renders a list of books using the <Book> component and the book component Takes the book data as a prop and renders the title and the author
you can add a button for removing books, its not suppose to work now.
- [ ] please there should be a form that allows the creation of a new book. with inputs for the title and the author of the book, you can add a submit button. it should not work yet, this is just a placeholder for now (this can be done in the newbookform component if you wish)

see project guidelines. you can go through it again 👇🏾 
![Screenshot (944)](https://user-images.githubusercontent.com/65298048/230488941-815eb258-cbca-4b5c-bf4b-ece48c76761c.png)